projects/admin/src/app/record/detail-view/item-detail-view/item-transaction/item-transaction.component.html

```diff
@@ -0,0 +1,37 @@
+<!--
+  RERO ILS UI
+   Copyright (C) 2019 RERO
+  
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, version 3 of the License.
+  
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU Affero General Public License for more details.
+  
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container *ngIf="transaction">
+  <div class="row">
+    <div class="col-5" *ngIf="transaction.metadata.patron_pid | getRecord:'patrons' | async as patron">
+      <a [routerLink]="['/circulation', 'checkinout']" [queryParams]="{ patron: patron.metadata.barcode }">
+        {{ patron.metadata.last_name }} {{ patron.metadata.first_name }} ({{ patron.metadata.barcode }})
+      </a>
+    </div>
+    <div class="col-4" *ngIf="'loan_borrow' === type">
+      {{ transaction.metadata.extension_count ?  transaction.metadata.extension_count : '-' }}
+    </div>
+    <div class="col-4" *ngIf="'loan_request' === type">
+      {{ transaction.metadata.pickup_location_pid | getRecord:'locations':'field':'pickup_name' | async }}
+    </div>
+    <div class="col-3" *ngIf="'loan_borrow' === type">
+      {{ transaction.metadata.end_date | dateTranslate:'longDate' }}
+    </div>
+    <div class="col-3" *ngIf="'loan_request' === type">
+        {{ transaction.metadata.transaction_date | dateTranslate:'longDate' }}
+      </div>
```

Comment:
Cosmetic : Limit "IF" statement (my point of view)
```html
<ng-container *ngIf="type === 'loan_borrow'">
  <div class="col-4">
      {{ transaction.metadata.extension_count ?  transaction.metadata.extension_count : '-' }}
  </div>
  <div class="col-3">
      {{ transaction.metadata.end_date | dateTranslate:'longDate' }}
  </div>
</ng-container>
<ng-container *ngIf="type === 'loan_request'">
  <div class="col-4">
      {{ transaction.metadata.pickup_location_pid | getRecord:'locations':'field':'pickup_name' | async }}
  </div>
  <div class="col-3">
      {{ transaction.metadata.transaction_date | dateTranslate:'longDate' }}
  </div>
</ng-container>
```

Reply:
done